portfolio/src/main/webapp/index.html

```diff
@@ -167,12 +168,21 @@ <h3>Shpe</h3>
            Click To Change Color 
         </button>
     </div>
+    -->
```

Comment:
I see what's happening. You're commenting out the majority of your site so we can go straight to the fetch, but you don't want to delete it. I'd say leave it in (they can both be there) or delete it entirely. It will still be in the git history, so you won't lose it.

Reply:
I'd say if you do leave it, perhaps add a comment alongside it explaining why the code is commented out. This sort of practice helps future coders and reviewers

Reply:
yea I didn't know whether I should delete my old portfolio code or not.